### DISCUSSION TITLE
Mouse and keyboard button press emulation with Clickermann inside Sandboxie

### DISCUSSION THREAD

**Discussion by FCPerfectO** [sandboxie-plus/Sandboxie#2115] [General]
Hello,

The "Allow programs in this sandbox to emulate keyboard and mouse input" setting was in the very old version 3.62 of Sandboxie, for example. but this version does not run on Win10. working new versions of Sandboxie already without this setting. please help me to correctly configure the latest version of Sandboxie (or any other sandbox that will hide Clickermann from being detected by other programs in the system) so that emulation of keystrokes on the keyboard and mouse works.

When I start script in Clickermann:

`go:
WAIT(RND(1,5)) // случайная задержка в сек
LCLICK(RND(614,667),RND(504,512))
WAIT(RND(1,2))
LCLICK(RND(614,667),RND(504,512))
//WAIT(RND(260,275))
WAIT(RND(1,4))
LCLICK(RND(57,78),RND(422,442))
WAIT(RND(21,24))
LCLICK(RND(57,78),RND(422,442))
WAIT(RND(1,3))
LCLICK(RND(57,78),RND(422,442))
WAIT(RND(1,3))
goto(go)`

Cursor move to area RND(614,667),RND(504,512), press left mouse button DOWN, but don't press it UP and don't move second time to area RND(614,667),RND(504,512) and nothing happens.

If I use another script with separate LDOWN and LUP - problem is the same - Clickerman press only LDOWN and don't relise left button with LUP and cursor don't move.

`WAIT(2)
LDOWN(38,45)
LUP(38,45)
WAIT(3)
LDOWN(600,350)
LUP(600,350)`

Help me please.

**Comment by isaak654** [comment]
> The "Allow programs in this sandbox to emulate keyboard and mouse input" setting was in the very old version 3.62 of Sandboxie, for example.

That option was discontinued since Sandboxie v4, so it's not usable anymore: [BlockFakeInput](https://sandboxie-plus.github.io/sandboxie-docs/Content/BlockFakeInput.html) 

More settings can be found in the [changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md) notes with a simple search of the `=` sign in your browser. Unfortunately there are not many people available to document some of them in [sandboxie-docs](https://github.com/sandboxie-plus/sandboxie-docs), even if they can earn an unrestricted Sandboxie certificate.

### EDIT: See https://github.com/sandboxie-plus/Sandboxie/issues/2117 for the continuation.
